mypy/fastparse2.py

```diff
@@ -346,23 +349,23 @@ def set_type_optional(self, type: Type, initializer: Expression) -> None:
         if isinstance(type, UnboundType):
             type.optional = optional
 
-    def transform_args(self, n: ast27.arguments, line: int) -> List[Argument]:
+    def transform_args(self, n: ast27.arguments, line: int) -> Tuple[List[Argument], List[Statement]]:
         # TODO: remove the cast once https://github.com/python/typeshed/pull/522
```

Comment:
I have a different PR which cleans up all the redundant casts and unused ignores (but that's waiting on the fast parser becoming the default).